actions/slack-payload/src/libs/workflow-context/workflow-context.ts

```diff
@@ -90,20 +96,19 @@ const mapAttachmentsData = (attachments: ListWorkflowRunArtifacts): AttachmentsD
     {
       buildLogsArtifactId: 0,
       testLogsArtifactId: 0,
-      environmentArtifactId: 0,
+      workflowInfoArtifactId: 0,
     }
   );
 };
 
-const getEnvironmentAttachmentId = (attachments: ListWorkflowRunArtifacts): number => {
-  const environmentAttachment = attachments.artifacts.find(
-    (artifact) => artifact.name === 'environment'
+const getWorkflowInfoAttachmentId = (attachments: ListWorkflowRunArtifacts): number => {
+  const workflowInfoAttachment = attachments.artifacts.find(
+    (artifact) => artifact.name === 'workflowInfo'
   );
-  return environmentAttachment ? environmentAttachment.id : 0;
+  return workflowInfoAttachment ? workflowInfoAttachment.id : 0;
 };
 
-const mapPullRequestData = (pullRequest: PullRequest | undefined): PullRequestData | undefined => {
-  if (!pullRequest) return;
+const mapPullRequestData = (pullRequest: PullRequest): PullRequestData | undefined => {
```

Comment:
This is never going to return undefined
```suggestion
const mapPullRequestData = (pullRequest: PullRequest): PullRequestData => {
```